src/index.ts

```diff
@@ -57,6 +59,7 @@ ${chalk.bold('Options:')}
     --clean             Clear out the destination directory before install.
     --optimize          Transpile, minify, and optimize installed dependencies for production.
     --babel             Transpile installed dependencies. Also enabled with "--optimize".
+    --entry             Auto-detect imports from file(s). Supports glob.
     --strict            Only install pure ESM dependency trees. Fail if a CJS module is encountered.
     --no-source-map     Skip emitting source map files (.js.map) into dest
```

Comment:
From our discussions, I left out `--dep-tree` as an option. Yes, it would be useful for building a “pika bundle analyzer” in the future, but first it should be a discussion on if that’s valuable, and if so what that should look like. Until then it doesn’t provide much value.